shared-libs/transitions/src/config.js

```diff
@@ -0,0 +1,13 @@
+let config = {},
+    translations = {};
+
+module.exports = {
+  init: (c, t) => {
+    config = Object.assign({}, c);
+    translations = Object.assign({}, t);
```

Comment:
Do you need to shallow clone the translations? Will this still be updated if the translations are modified?

Reply:
I'm calling `init` in both API and Sentinel whenever translations or updated. 
-> https://github.com/medic/medic/blob/4857-detached-sentinel-bis/api/src/config.js#L173
-> https://github.com/medic/medic/blob/4857-detached-sentinel-bis/sentinel/src/config.js#L48

I can remove the shallow clone.

Reply:
Ahh I must've missed that. Sorry.